File: SWA/day1/Task_2/CBO/LowCBO.java
class SimplePayment {
    void pay() {
        System.out.println("Simple payment done.");
    }
}

class SnackCounter {
    SimplePayment payment = new SimplePayment();

    void buySnack() {
        payment.pay();
        System.out.println("Snack purchased.");
    }
}

public class LowCBO {
    public static void main(String[] args) {
        SnackCounter counter = new SnackCounter();
        counter.buySnack();
        System.out.println("Low CBO = 1 dependency");
    }
}

File: SWA/day1/Task_2/CBO/lowcbo.ts
class SimplePayment {
  pay() { console.log("Simple payment done."); }
}

class SnackCounter {
  payment = new SimplePayment();

  buySnack() {
    this.payment.pay();
    console.log("Snack purchased.");
  }
}

export function lowcbo() {
  const counter = new SnackCounter();
  counter.buySnack();
  console.log("Low CBO = 1 dependency");
}

lowcbo();
